src/main/resources/views/includes/template.ftl

```diff
@@ -80,6 +80,11 @@
                             <a href="${basePath}/${clusterId}/group"><i class="fa fa-fw fa-object-group" aria-hidden="true"></i> Consumer Groups</a>
                         </li>
                     </#if>
+                    <#if roles?seq_contains("user") == true>
+                        <li class="${(tab == "user")?then("active", "")}">
+                            <a href="${basePath}/${clusterId}/user"><i class="fa fa-fw fa-users" aria-hidden="true"></i> Users</a>
```

Comment:
Rename Globally Users with ACLS please to be consistent with Kafka Naming

Reply:
Just to be dure, can you be more specific on this one ?

- Should I rename the "Users" section in the sidenav to ACLS ? 
- Should I rename roles, class and variables ?
- Should I change the "user" path to "acls" path ?

Reply:
Sorry don't see it :-1  and thanks for pointing this :+1: 
 
url / sidebar / role / ClassName & Vars. 

I think there is will be too many people that will mix up with KafkaHQ user.
Since in KafkaHQ, we use the term role, it will be easier to understand I think